web/viewer.html

```diff
@@ -298,6 +312,19 @@
 
                 <div class="verticalToolbarSeparator hiddenSmallView"></div>
 
+                <div id="editorModeButtons" class="splitToolbarButton toggled hidden" role="radiogroup">
+                  <button id="editorNone" class="toolbarButton toggled" title="Disable Annotation Editing" role="radio" aria-checked="true" tabindex="31" data-l10n-id="editor_none">
+                    <span data-l10n-id="editor_none_label">Disable Editing</span>
+                  </button>
+                  <button id="editorFreeText" class="toolbarButton" title="Add FreeText Annotation" role="radio" aria-checked="false" tabindex="32" data-l10n-id="editor_free_text">
+                    <span data-l10n-id="editor_free_text_label">FreeText Annotation</span>
+                  </button>
+                  <button id="editorInk" class="toolbarButton" title="Add Ink Annotation" role="radio" aria-checked="false" tabindex="33" data-l10n-id="editor_ink">
+                    <span data-l10n-id="editor_ink_label">Ink Annotation</span>
+                  </button>
+                  <div class="verticalToolbarSeparator hiddenSmallView"></div>
```

Comment:
I don't think you want hiddenSmallView here, since I'm assuming that you probably always want the editing-buttons visible in the main toolbar (when they're enabled).
```suggestion

                  <div class="verticalToolbarSeparator"></div>
```